src/style.css

```diff
@@ -0,0 +1,40 @@
+.body {
+  margin: 0;
+  box-sizing: border-box;
+  font-size: 0;
+}
+
+.container {
+  display: flex;
+  flex-flow: column wrap;
+}
+
+.star__star {
+  display: inline-block;
+  margin: 2px 6px 2px 2px;
+  height: 12px;
+  width: 12px;
+
+  background-image: url(./images/star.svg);
+  background-size: cover;
+}
+
+.stars--1 :nth-child(1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n+2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 :nth-child(-n+3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 :nth-child(n) {
```

Comment:
if i add more stars to each row, let's say 7, this modifier will not make 5 stars colored yellow, it will make all of them yellow, which might be confusing